Source/Core/DiscIO/NANDImporter.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+namespace DiscIO
+{
+class NANDImporter final
+{
+public:
+  static NANDImporter& GetInstance()
```

Comment:
To be honest, I just copied the logic from `NANDContentLoader.h`, because I thought installing a WAD was very similar to importing a NAND. Do you have suggestions to what I should change it to?